Classes/Neos/Neos/Ui/Domain/Model/AbstractChange.php

```diff
@@ -95,13 +95,13 @@ protected function reloadDocument()
 
     /**
      * Inform the client that a node has been created, the client decides if and which tree should react to this change.
-     *
+     * @param NodeInterface $subject
      * @return void
      */
-    protected function addDocumentNodeCreatedFeedback()
+    protected function addDocumentNodeCreatedFeedback($subject = null)
     {
+        $node = $subject ?: $this->getSubject();
         $nodeService = new NodeService();
-        $node = $nodeService->getClosestDocument($this->getSubject());
```

Comment:
@grebaldi why was this `getClosestDocument` necessary? The method is called `addDocumentNodeCreatedFeedback`, so I assume it should only create feedback for document nodes, no?